### ISSUE TITLE
Need to develop initial launch screen

### ISSUE THREAD

**Issue by abw1914** [abw1914/EncoreLinkV2#2]
Since there are two separate logins for musicians and facility owners - should users identify the corresponding login on the launch screen.

**Comment by jqn** [comment]
@abw1914 I think the initial screen should be the login screen. We have duplicated functionality here with the login screen. This should be a splash screen with the encorelink logo only that shows only briefly when the app is opened.

**Comment by BenMorrisRains** [comment]
@jqn I agree. I think that once the user is logged in, they should be directed to the correct views based on their role. I find that the idea of having a switch button like in the wireframes is cool, but it admittedly confused me upon first seeing it on what its job was.

**Comment by baghaii** [comment]
Though the first screen is called SplashScreenActivity, it is actually the login screen. @abw1914 was creating activities that he felt needed to exist without really filling them in and using non-standard naming conventions for stuff. 

@raemacias created the existing version of the login activity that is called SplashScreenActivity. We should probably rename it to reduce confusion.

The switch in the wireframe does not exist anywhere yet. The wireframes were all my suggestions, and we do not have to stick to keeping them the way they are. We can change them. I created them as a point of discussion so that we can divide up the work more easily so we can move forward with the project instead of guessing what we need to do next. I created them by looking at the web version of the project and including most of the functionality. There were probably a few fields I left off because they didn't make sense to me, or they didn't fit on the screen. 

The reason I created the switch was that I just wasn't sure that we wanted to create a separate activity for people to choose whether they are a musician or a facility manager and then go to two screens that are either identical or exactly the same to log in. Other forms of toggle button could be used here for clarity if we don't want to create a separate activity for choosing the type of user that is logging in.
